npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher.rb

```diff
@@ -221,6 +222,15 @@ def lockfiles
         }
       end
 
+      sig { returns(T::Hash[Symbol, T.nilable(Dependabot::DependencyFile)]) }
+      def rcfiles
```

Comment:
can we use more explicit names for the functions please?

Reply:
I will check but since this is similar to lockfiles returning npmrc, yarnrc files that's why I made it rcfiles. I will try to make it better also add comment.